declarations/Canva and Affinity.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "Canva and Affinity",
```

According to our documentation, we would need to attribute this to only one service. The one responsible for the policies. In this case `Canva`. This is the Service Name. But also it usually has the same value as the Service ID which is the name of the JSON file. So we might need to change that as well to `Canva` 

```suggestion
  "name": "Canva",
```